### DISCUSSION TITLE
PWAsForFirefox 1.3.0

### DISCUSSION THREAD

**Discussion by filips123** [filips123/PWAsForFirefox#97] [Announcements]
## News

To avoid infringing Mozilla trademark policies and confusing users that this project is affiliated with Firefox and Mozilla, FirefoxPWA has been renamed to "Progressive Web Apps for Firefox" (shortly: "PWAsForFirefox" or "pwas-for-firefox"). Please use one of those names when referring to this project.

The project logo and all related icons have also been changed to use the CC0-licenced fox icon instead of the official Firefox logo.

The repository URL has been changed to *<https://github.com/filips123/PWAsForFirefox>* and the AMO URL will be changed to *<https://addons.mozilla.org/firefox/addon/pwas-for-firefox/>* soon after this release. Existing packages and GitHub links will probably continue to work as GitHub provides automatic redirects from the old repository URL, but all links to AMO page need to be changed.

*Note:* Unfortunately, this is still not everything I planned for that bigger update. I will hopefully have time to release it this year...

## Added

* Support for using this project on platforms without official Firefox support.
* Automatic pre-built DEB and RPM packages for Linux on ARM.

## Changed

* Rebranded the project as Progressive Web Apps for Firefox (PWAsForFirefox).
* Improved the runtime installation step is reflect added support for additional platforms.

<hr /><em>This discussion was created from the release <a href='https://github.com/filips123/PWAsForFirefox/releases/tag/v1.3.0'>PWAsForFirefox 1.3.0</a>.</em>

**Comment by inwardgarden** [comment]
After updating to 1.3.0 (using Firefox 95 on Win 11), I see the following (apps and profiles missing):

[screenshot](https://i.imgur.com/kOp7Ybc.png)

I can open an issue if desired. 

Tried reinstalling the runtime with no change. Tried going back to previous release, and it appears to relink things back up- but then I get the forced popup to upgrade anytime I click the extension icon. Uninstalling everything and reinstalling also did not help. If I do it from scratch, it fails to download the runtime through the setup steps, with the same error.

**Reply by filips123** [reply]
Can you please check [troubleshooting steps](https://github.com/filips123/PWAsForFirefox/wiki/Troubleshooting) and provide logs?

Maybe I renamed something that I shouldn't when I renamed the project.

**Reply by inwardgarden** [reply]
Issue #98 has been opened by another user- I'll post my current info + any updates from later today I have under there.
